README.md

```diff
@@ -23,21 +24,6 @@ Download the SDK using npm or yarn
 npm install -s @hubspot/calling-extensions-sdk
 ```
 
-#### Running the demo Calling Extensions Widget project
-
-##### Run the demo widget project
-
-```shell
-# install npm build dependencies and start the demo project
-# The following commands will automatically start the browser on the demo page.
-cd /demo
-npm start
-```
-
-##### Launch the demo widget from HubSpot
-
-Navigate to the a contact/company page in HubSpot and launch calling, the demo widget should load inside an iFrame.
-
```

Comment:
This information really needs the context of adding the localStorage information to view it in HubSpot.